tests/src/main/java/net/neoforged/neoforge/debug/crafting/IngredientTests.java

```diff
@@ -66,6 +68,37 @@
 
 @ForEachTest(groups = "crafting.ingredient")
 public class IngredientTests {
+    @GameTest
+    @EmptyTemplate
+    @TestHolder(description = "Tests if BlockTagIngredient works")
+    static void blockTagIngredient(final DynamicTest test, final RegistrationHelper reg) {
+        reg.addProvider(event -> new RecipeProvider(event.getGenerator().getPackOutput(), event.getLookupProvider()) {
+            @Override
+            protected void buildRecipes(RecipeOutput output) {
+                ShapelessRecipeBuilder.shapeless(RecipeCategory.MISC, Items.MUD)
+                        .requires(new BlockTagIngredient(BlockTags.CONVERTABLE_TO_MUD).toVanilla())
```

Comment:
Sorry I missed this in my previous review: Can you wrap the ingredient in a `new TestEnabledIngredient(`? (See example below).